reactive/load_balancer.py

```diff
@@ -240,38 +242,46 @@ def provide_application_details():
             address = vips
         elif dns_record:
             address = dns_record
-    if address:
-        website.configure(port=hookenv.config('port'),
-                          private_address=address,
-                          hostname=address)
+        else:
+            address = hookenv.unit_get('public-address')
     else:
-        website.configure(port=hookenv.config('port'))
+        address = hookenv.unit_get('public-address')
+    return address
+
+
+@when('endpoint.lb-consumers.joined',
+      'leadership.is_leader')
+def provide_lb_consumers():
+    '''Respond to any LB requests via the lb-consumers relation.
+
+    This is used in favor for the more complex two relation setup using the
+    website and loadbalancer relations going forward.
+    '''
+    lb_consumers = endpoint_from_name('lb-consumers')
+    address = _get_lb_address()
+    for request in lb_consumers.all_requests:
+        request.response.address = address
+        lb_consumers.send_response(request)
```

Comment:
Without hacluster or loadbalancer-ips configured, this will tell other units to connect to a public-address, which is not ideal.

It's not obvious what should be used instead here. We could use the ingress address of the lb-consumers relation, but that might not work for every client - what if the consumer is in the same model as us, but one of the clients of the consumer is connected via cross-model relation?